static/img/Nyxx_Logo.svg

```diff
@@ -0,0 +1,12 @@
+<path d="M94.6697 137.481C94.6697 136.281 95.6081 135.309 96.7628 135.309C115.614 135.309 125.998 135.309 144.849 135.309C146.004 135.309 146.942 136.281 146.942 137.481V200.494V263.506C146.942 264.706 146.006 265.679 144.852 265.679H96.7606C95.6059 265.679 94.6697 264.706 94.6697 263.506V137.481Z" fill="#5865F2"/>
+<path d="M110.074 174.233C109.257 173.385 109.259 172.008 110.075 171.159C122.011 158.755 131.771 148.613 143.707 136.21C144.523 135.361 145.848 135.36 146.665 136.209L192.333 183.666L234.082 227.051C234.899 227.9 234.899 229.276 234.082 230.124L200.449 265.076C199.632 265.925 198.308 265.925 197.492 265.076L110.074 174.233Z" fill="#5865F2"/>
+<path d="M197.663 137.481C197.663 136.281 198.602 135.309 199.756 135.309C218.608 135.309 228.992 135.309 247.843 135.309C248.998 135.309 249.936 136.281 249.936 137.481V200.494V263.506C249.936 264.706 249 265.679 247.845 265.679H199.754C198.599 265.679 197.663 264.706 197.663 263.506V137.481Z" fill="#5865F2"/>
+<path d="M345.579 263.506C345.579 264.706 344.643 265.679 343.488 265.679C323.973 265.679 304.458 265.679 284.942 265.679C283.788 265.679 282.851 264.706 282.851 263.506V144.513C282.851 142.22 285.826 141.49 286.813 143.541L312.345 196.607C313.115 198.208 315.315 198.208 316.085 196.607L341.618 143.541C342.604 141.49 345.579 142.22 345.579 144.513V263.506Z" fill="#5865F2"/>
+<path d="M367.965 190.595C367.149 189.746 367.153 188.365 367.968 187.515C378.817 176.218 384.907 169.876 395.756 158.579C396.572 157.729 397.898 157.726 398.713 158.575L434.135 195.461L469.556 232.347C470.372 233.196 470.372 234.574 469.556 235.423L441.762 264.367C440.946 265.216 439.623 265.216 438.808 264.367L367.965 190.595Z" fill="#5865F2"/>
+<path d="M398.713 264.367C397.898 265.216 396.572 265.213 395.756 264.363C384.907 253.066 378.817 246.724 367.968 235.427C367.152 234.577 367.149 233.196 367.965 232.347L403.386 195.461L438.807 158.575C439.623 157.726 440.946 157.726 441.761 158.575L469.556 187.519C470.372 188.368 470.372 189.746 469.556 190.595L398.713 264.367Z" fill="#5865F2"/>
+<path d="M493.42 190.595C492.604 189.746 492.607 188.365 493.423 187.515C504.272 176.218 510.362 169.876 521.211 158.579C522.027 157.729 523.353 157.726 524.168 158.575L559.59 195.461L595.011 232.347C595.827 233.196 595.827 234.574 595.011 235.423L567.216 264.367C566.401 265.216 565.078 265.216 564.262 264.367L493.42 190.595Z" fill="#5865F2"/>
+<path d="M524.168 264.367C523.352 265.216 522.026 265.213 521.21 264.363C510.361 253.066 504.272 246.724 493.423 235.427C492.607 234.577 492.603 233.196 493.419 232.347L528.841 195.461L564.262 158.575C565.078 157.726 566.4 157.726 567.216 158.575L595.011 187.519C595.826 188.368 595.826 189.746 595.011 190.595L524.168 264.367Z" fill="#5865F2"/>
+</svg>
```

Comment:
Would have to check if this is the logo we go forward with

Reply:
Yeah, no problem, I just created this logo for testing purposes.
I can ask my friend if she's okay to create a "real" logo for us, I'm not good at this kind of stuff 😅

Reply:
Looks pretty cool. `y` could look more like y and would good with me

Reply:
Okay, I'll see what I can do!

Reply:
So, is this better now?